### ISSUE TITLE
Accessibility: WCAG 2.4.5 Multiple Ways Level AA

### ISSUE THREAD

**Issue by carolinan** [WordPress/twentytwentyone#79]
More than one way is available to locate a Web page within a set of Web pages except where the Web Page is the result of, or a step in, a process.

Reference: https://www.w3.org/TR/WCAG21/#multiple-ways

- [x] Menu **-Optional**
- [ ] Next and previous post navigation, links between paginated posts (page breaks), link to parent on attachment page (image.php)
- [x] Search  **Optional with widget or block**

@audrasjb do I understand the criteria correctly that this will fail because both the menu and search is optional?
There is no fall back page menu, so if no menu is set, there is no menu at all.

Even though 5.5 added a sitemap, it is not a solution for this guideline since it is not really human readable nor helpful for screen reader users, since the link name is just the file name and the slug.

**Comment by carolinan** [comment]
The above checkbox that is not checked yet, is mainly about testing the image.php attachment page.
